packages/react-ui/storybook/stories/atoms/Typography.stories.js

```diff
@@ -28,6 +28,33 @@ const options = {
           'code3'
         ]
       }
+    },
+    weight: {
+      control: {
+        type: 'select',
+        options: ['regular', 'medium', 'strong']
+      }
+    },
+    italic: {
+      defaultValue: false,
+      control: {
+        type: 'boolean'
+      }
+    },
+    color: {
+      description: 'Any theme color, default and custom ones',
+      control: {
+        type: 'select',
+        options: ['primary.dark', 'brand.navyBlue', 'default.main']
```

Comment:
Where is definition of all those colors available ?

(again, Typography.d.ts would be great)

Reply:
Are coming from the [theme](https://github.com/CartoDB/carto-react/blob/feature/design-system/packages/react-ui/src/theme/carto-theme.js) directly, you can use any color that we have declared there. 

This is just a small example to have in the story, just to know you can change the color via props in the Typography component.

Reply:
I guess that should be expressed in the typings

Reply:
color is common prop to Mui components (it's a [system property](https://mui.com/system/properties/#properties-reference-table))... do you want to restrict it to only theme colors? 

Right now, from what I've seen, it accepts any string. I mean, wheter a primaryText or a #000000, for example.

I think should be enough as it is, but let me know.

Reply:
Ok, now i see that in MUI5 it's much more robust than in MUI4. In MUI 4, only very few hardcoded colors were accepted ('primary', ...)

Thanks for explanation.